packages/fxa-auth-server/lib/customs.js

```diff
@@ -257,6 +282,84 @@ class CustomsClient {
       });
     }
   }
+
+  // #region Customs V2
+
+  /**
+   * Version 2 Customs Approach
+   * =======================================================================================
+   * This uses a library provided by libs and works directly with Redis to make rate limiting
+   * decisions. The previous customs check to see if there is 'new' configuration for the
+   * customs action being checked. If there is, we will call into this code instead of calling
+   * the legacy customs service.
+   */
+  async checkV2(request, type, action, opts) {
+    // Short circuit if rate limit wasn't provided.
+    if (this.rateLimit == null) {
+      return false;
+    }
+
+    // Fallback to the legacy customs service approach, if v2 action isn't configured
+    const actionConfigured = this.rateLimit.supportsAction(action);
+    if (!actionConfigured) {
+      this.statsd.increment(`${serviceName}.check.v1`, [`action:${action}`]);
+      return false;
+    }
+
+    // Otherwise, call the new nx lib instead of the legacy service
+    this.statsd.increment(`${serviceName}.check.v2`, [`action:${action}`]);
+    const result = await this.rateLimit.check(action, opts);
+
+    // If statsd was provided, record metrics
+    if (this.statsd) {
```

Comment:
I see two statsd calls above.  Should those be wrapped in an `if`?

Reply:
Thanks